tools/src/commands/CommentatorCommand.ts

```diff
@@ -0,0 +1,66 @@
+import { Command } from '@expo/commander';
+import chalk from 'chalk';
+import { link } from '../Formatter';
+
+import { commentOnIssueAsync } from '../GitHubActions';
+import logger from '../Logger';
+
+type ActionOptions = {
+  payload: string;
+};
+
+export type CommentatorComment = {
+  issue: number;
+  body: string;
+};
+
+export type CommentatorPayload = CommentatorComment[];
+
+export default (program: Command) => {
+  program
+    .command('commentator')
+    .alias('comment')
+    .option(
+      '-p, --payload <payload>',
+      'Serialized and escaped JSON describing what and where to comment.'
```

Comment:
```suggestion
      'Serialized and escaped JSON array describing what and where to comment.'
```
You are throwing if it's not the array 👅 
Maybe add some example regarding how this payload should look like? 🤔

Reply:
Regarding example: I think it's too difficult to use this command locally — right now it's intended to be run only on the CI so I don't think it needs to be explanatory (you can still take a look at type definitions). In the future, if there will be a need, I would add separate flags to make it more convenient to use locally, instead of having to serialize and escape the payload properly.
Anyway, I've added a short example to command's description.